### PR TITLE
Corrected link to medley introduction

### PR DESCRIPTION
**Pull request by neauoire**

### CODE REVIEW

**Comment by masinter** [comment]
@stumbo no wiki

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Maiko is the implementation of the Medley Interlisp virtual machine, for a
 byte-coded Lisp instruction set and some low-level functions for
 connecting with Lisp for access to display (via X11) and disk etc.
 
-For an overview, see [Medley Interlisp Introduction](https://github.com/Interlisp/medley/wiki/Medley-Interlisp-Introduction).
+For an overview, see [Medley Interlisp Introduction](https://github.com/Interlisp/medley/wiki/Introduction).
```
Comment (stumbo) [inline README.md:7]: We are in the process of deprecating the wiki page and centralizing all our documentation on our web site, [Interlisp.org](https://interlisp.org/).

A better place to link to may be:  [https://interlisp.org/medley/using/docs/medley/](https://interlisp.org/medley/using/docs/medley/)

The website is a work in progress - suggestions and improvements there are always appreciated.

**Review by stumbo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (neauoire) [reply README.md:7]: Thanks! Yes that's the page I was looking for :)

**Comment by stumbo** [comment]
Close this PR - the author marked the comment as resolved but never updated the file.  Created PR #484 with the correct location and opened a PR for it.

**Pull request closed without merging**